app/uk/gov/hmrc/vatsubscription/models/PendingChanges.scala

```diff
@@ -35,8 +35,7 @@ object PendingChanges {
   private val bankDetailsPath =  __ \ "bankDetails"
   private val returnPeriodPath = __ \ "returnPeriod"
   private val mandationStatusDesPath = __ \ "mandationStatus" \ "mandationStatus"
-  private val mandationStatusWritesPath = __ \ "mandationStatus"
-  private val commsPreferencePath = __ \ "commsPreference"
+  private val commsPreferenceDesPath = __ \ "commsPreference" \ "commsPreference"
```

Comment:
I'm a little worried these changes are showing up as they were the changes I made in the bug fix which are now on master. I'm not sure the rebase has been done properly